ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/core/Transaction.java

```diff
@@ -494,7 +494,7 @@ public Transaction signAndBuild(final SECP256K1.KeyPair keys) {
       return build();
     }
 
-    protected SECP256K1.Signature computeSignature(final SECP256K1.KeyPair keys) {
+    SECP256K1.Signature computeSignature(final SECP256K1.KeyPair keys) {
```

Comment:
Doesn't look like you need to expose this.

Reply:
Ok